rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/creation/CreateNodeSourceWindow.java

```diff
@@ -54,11 +45,8 @@
  */
 public class CreateNodeSourceWindow extends NodeSourceWindow {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.smartgwt.client.widgets.form.fields.TextItem'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)